src/test/java/com/google/api/codegen/testdata/nodejs/nodejs_multiple_services.baseline

```diff
@@ -130,21 +130,17 @@ const gapic = Object.freeze({
  * @module {object} @google-cloud/multiple-services
  * @alias nodejs-multiple-services
  *
- * @example <caption>Install the client library with
- *   <a href="https://www.npmjs.com/">npm</a>:</caption>
+ * @example <caption>Install the client library with <a href="https://www.npmjs.com/">npm</a>:</caption>
  * npm install --save @google-cloud/multiple-services
  *
  * @example <caption>Import the client library:</caption>
  * const multiple-services = require('@google-cloud/multiple-services');
```

Comment:
btw `const multiple-services = require('@google-cloud/multiple-services');` is not valid javascript. The variable needs to be changed to camelCase.